runtime/cere-dev/src/lib.rs

```diff
@@ -149,7 +149,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 54112,
+	spec_version: 54113,
```

Comment:
You need to increase `spec_version` in both `cere` and `cere_dev` runtimes as you affected both of them by making changes to the pallet